cql3/type_json.cc

```diff
@@ -228,6 +230,18 @@ struct from_json_object_visitor {
     }
     bytes operator()(const boolean_type_impl& t) {
         if (!value.IsBool()) {
+            if (value.IsString()) {
+                std::string str(rjson::to_string_view(value));
+                boost::trim_all(str);
```

Comment:
I'm curious - since Cassandra evidently doesn't do this trim_all() - you mentioned that Cassandra doesn't allow whitespace here - why do we want to support this? It doesn't look to me like a very useful addition to the already bizarre feature which only half-works in Cassandra (works just in prepared statements).
On the other hand, maybe you're right and since it's so easy to do (just this trim_all call), we can just do it...

Reply:
I did this exactly for this reason you mentioned, it was easy and I thought it would be nice :)
Moreover, this works also in unprepared statements in Cassandra. There was bug in test, `fromJson()` was missing and the test was trying to insert string into boolean column.

Reply:
Ok, this trimming doesn't sound too useful or nice to me, but I don't hate it either, so I'll accept it.